### ISSUE TITLE
nuitka.bat will not work if we install nuitka in a different folder

### ISSUE THREAD

**Issue by ArkadiuszMichalski** [Nuitka/Nuitka#1685]
When we change PiP instalation target via `PIP_TARGET` then [nuitka.bat](https://github.com/Nuitka/Nuitka/blob/develop/misc/nuitka.bat) will not work. This file check only two `python.exe` locations related to the `nuitka.bat` location. Would be nice if this file would:

- firstly check some variable for python location
- or check exist also for second case and if not exist then add another `else` with invoke just `python args`, which will be searched in setting `PATH`.

**Comment by ArkadiuszMichalski** [comment]
Yes, that's enough. We must remember to have the `bin\nuitka.bat` and `python.exe` in the PATH variable before run this file.

**Comment by kayhayen** [comment]
The 1.0 release was done already.
